rest-api/src/com/db/capella/integration/WorkspaceProjectInt.java

```diff
@@ -0,0 +1,100 @@
+// Copyright DB InfraGO AG and contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package com.db.capella.integration;
+
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.ui.IEditorPart;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPage;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.part.FileEditorInput;
+
+import com.db.capella.model.WorkspaceProject;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.ALWAYS)
+public class WorkspaceProjectInt extends WorkspaceProject {
+  org.eclipse.core.resources.IProject obj;
+
+  public WorkspaceProjectInt(org.eclipse.core.resources.IProject obj) {
+    super();
+    this.obj = obj;
+    ReflectionUtils.copyProperties(obj, this);
+    this.computeModified();
+  }
```

Comment:
It's hard to understand why this is needed or what it's intended for. My assumption is that you need to cast `WorkspaceProjectInt` to `IProject`. But why don't you create subclass of IProject?